Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2519,14 +2524,18 @@ def fetch_incidents():
     incidents: list = []
     detections: list = []
     idp_detections: list = []
+    iom_config_incidents: list[dict[str, Any]] = []
+    ioa_event_incidents: list[dict[str, Any]] = []
     last_run = demisto.getLastRun()
     demisto.debug(f'CrowdStrikeFalconMsg: Current last run object is {last_run}')
     if not last_run:
-        last_run = [{}, {}, {}]
+        last_run = [{}, {}, {}, {}, {}]
     last_run = migrate_last_run(last_run)
     current_fetch_info_detections: dict = last_run[0]
     current_fetch_info_incidents: dict = last_run[1]
     current_fetch_info_idp_detections: dict = {} if len(last_run) < 3 else last_run[2]
+    current_fetch_info_iom_configs: dict = {} if len(last_run) < 4 else last_run[3]
```

Comment:
The integration fetches 5 incident types. At first, it was only 2, but a recent enhancement was added that added 1 incident type, and I introduced two new incident types, therefore, the last run object will be a list of 5 dictionaries, and I believe the whole **{} if len(las_run) < length else last_run[length - 1]** was to safely handle the last run objects of the newly added incident types.

Reply:
differentiating types according to the length of the output sounds dangerous - what if we add an output to one of them in the future, or add a new one with 3/4/5 outputs? 

I know this was not your decision and was there before, but can we find some way to avoid at least part of the problem?

Reply:
This happens after `migrate_last_run`, so we can be sure that this index exists in the list